### ISSUE TITLE
Missing links on "My Calendar" page

### ISSUE THREAD

**Issue by jean-gui** [w3c/calendar#105]
There are no links to past events and export section on https://www.w3.org/users/myprofile/calendar/ thus making them hard to find.

We should add links similar to those on group pages https://www.w3.org/groups/wg/epub/calendar/.
